<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Welcome</class>
 <widget class="QMainWindow" name="Welcome">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1729</width>
    <height>1477</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SIC Assembler</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>1631</width>
      <height>1221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1631</width>
       <height>1083</height>
      </rect>
     </property>
     <attribute name="label">
      <string>** Source File</string>
     </attribute>
     <widget class="QPlainTextEdit" name="src_txt">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>1571</width>
        <height>1061</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1631</width>
       <height>1083</height>
      </rect>
     </property>
     <attribute name="label">
      <string>** List File</string>
     </attribute>
     <widget class="QPlainTextEdit" name="list_txt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1581</width>
        <height>1061</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <attribute name="label">
      <string>** Object FIle</string>
     </attribute>
     <widget class="QPlainTextEdit" name="obj_txt">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>1561</width>
        <height>1061</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1729</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionLoad_Srcfile"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAssemble">
    <property name="title">
     <string>Assembler</string>
    </property>
    <addaction name="actionAssemble"/>
    <addaction name="actionList_File"/>
    <addaction name="actionObject_File"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAssemble"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAssemble">
   <property name="text">
    <string>Assemble</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
  <action name="actionList_File">
   <property name="text">
    <string>List File</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
  <action name="actionObject_File">
   <property name="text">
    <string>Object File</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
  <action name="actionLoad_Srcfile">
   <property name="text">
    <string>Load Srcfile</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
